common.sh

```diff
@@ -16,7 +16,7 @@ f5m_configure() {
   PKG_CONFIG_PATH=$HOMEBREW_PREFIX/lib/pkgconfig cmake -B build -G Ninja \
     -DCMAKE_INSTALL_PREFIX=$HOMEBREW_PREFIX \
     -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_FIND_ROOT_PATH="/Library/Input Methods/Fcitx5.app/Contents;$HOMEBREW_PREFIX" \
+    -DCMAKE_FIND_ROOT_PATH="/Library/Input Methods/Fcitx5.app/Contents;$HOMEBREW_PREFIX;$CMAKE_FIND_ROOT_PATH" \
```

Comment:
I don’t think it’s needed. There is no such `CMAKE_FIND_ROOT_PATH`.

Reply:
It is used as an environment variable by librime.sh to pass marisa's build root to librime.

I will change it to a better name later.

Reply:
removed in favor of `CMAKE_PREFIX_PATH`